### DISCUSSION TITLE
Duckstation Core

### DISCUSSION THREAD

**Discussion by mattitotti** [OnionUI/Onion#644] [Q&A] [answered]
hi guys in the onion os there is the duckstation core for the ps1? if it isn't, there will be in the future?
thanks a lot .

**Comment by schmurtzm** [comment, marked as answer]
Hi, it's not possible on the Miyoo Mini, it requires hardware acceleration which is not supported by the Miyoo Mini.

**Reply by mattitotti** [reply]
thank you
